remind-bindings.el

```diff
@@ -5,7 +5,7 @@
 ;; Author: Mehmet Tekman
 ;; URL: https://github.com/mtekman/remind-bindings.el
 ;; Keywords: outlines
-;; Package-Requires: ((emacs "25.1"))
+;; Package-Requires: ((emacs "25.1") (omni-quotes "0.5") (popwin "1.0"½))
```

Comment:
Is the 1/2 symbol a typo

Reply:
Yes, it is typo. Sorry.
It seems that strange characters were entered because I was using a keyboard I did not usually use.